ba/react/react/react-todo-list/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import Context from "./context";
+import ToDoList from "./toDo/ToDoList";
+import AddToDo from "./toDo/addToDo";
+
+function App() {
```

Comment:
if you want to use functional component it is better to write const App =() => { const toggleToDo =() => {} and so on }